spec/opal/core/date_spec.rb

```diff
@@ -112,6 +112,13 @@
     end
   end
 
+  describe "#next_year" do
```

Comment:
Could you try to remove filters for `Date#next_year` and `Date#prev_year` in `spec/filters/bugs/date.rb`? If they pass, remove your specs please.
You can run all `Date` specs using `env RUBYSPECS=true PATTERN=spec/ruby/library/date/*_spec.rb bundle exec rake mspec_ruby_nodejs`

Reply:
I followed existing 4-year-old conventions for the specs for the other `Date#prev_*` and `Date#next_*` methods. If those conventions are wrong, I put these specs right next to the others so you can delete them all at once.

Reply:
> I followed existing 4-year-old conventions for the specs for the other Date#prev_* and Date#next_* methods

There's no such convention.

There are a lot of specs under `spec/opal/` that are just a copy of rubyspec's test suite. Initially they were copied (with small changes) because self-written parser (or compiler) was unable to parse/compile rubyspec specs. 

Ideally `spec/opal` directory should include only Opal-specific specs (like specs for `.JS` syntax or some edge case optimizations). We should review them and drop duplicates (definitely not in this PR).

Reply:
You just denied a convention exists and then explained with more words why that convention exists.